src/stream/src/common/table/state_table.rs

```diff
@@ -1251,7 +1251,7 @@ impl<S: LocalStateStore> StorageIterInner<S> {
     }
 
     /// Yield a row with its primary key.
-    #[try_stream(ok = (Vec<u8>, OwnedRow), error = StreamExecutorError)]
+    #[try_stream(ok = (Bytes, OwnedRow), error = StreamExecutorError)]
```

Comment:
Can we also use bytes key range in `StorageIterInner::new` and `StateStoreRead::iter`?

Reply:
I just had a try, and using bytes key range will cause a lot extra change. Given that this PR is already large, we may do it in a future PR.